modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/IgnitePdsDestroyCacheTest.java

```diff
@@ -394,4 +396,18 @@ public void cacheDestroyWithConcImplicitTx() throws Exception {
             // No op.
         }
     }
+
+    @Test
+    public void cleanupCacheDirectory() throws Exception {
+        IgniteEx ignite = startGrids(1);
+        ignite.cluster().state(ClusterState.ACTIVE);
+
+        ignite.createCache(new CacheConfiguration(DEFAULT_CACHE_NAME));
+        IgniteCache cache = ignite.cache(DEFAULT_CACHE_NAME);
+        cache.put(30000, 30000);
+
+        cache.destroy();
+
+        assertFalse(new File(((FilePageStoreManager)ignite.context().cache().context().pageStore()).workDir(), "cache-" + DEFAULT_CACHE_NAME).exists());
```

Comment:
This line seems too long

Reply:
And just in case, please assert that this dir exists before destroying the cache, so that there's no doubt that there are no error in test itself

Reply:
added additional assert